<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="10" hostname="LDCHOWN-58692" name="com.vcentry.lab.testCases.CheckoutPageValidator" tests="11" failures="0" timestamp="2023-05-07T12:35:03 IST" time="0.056" errors="1">
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="setUp" time="0.056" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <error type="java.io.FileNotFoundException" message="Source &#039;C:\Batch 256\J1_DDDTest\screenshot&#039; does not exist">
      <![CDATA[java.io.FileNotFoundException: Source 'C:\Batch 256\J1_DDDTest\screenshot' does not exist
at org.apache.commons.io.FileUtils.checkFileRequirements(FileUtils.java:1383)
at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1346)
at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1246)
at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1215)
at com.vcentry.lab.utils.Archiver.archiveScreenshot(Archiver.java:19)
at com.vcentry.lab.initializer.Initializer.initializer(Initializer.java:74)
at com.vcentry.lab.testCases.CheckoutPageValidator.setUp(CheckoutPageValidator.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
  <testcase name="validateCheckOutPage" time="0.000" classname="com.vcentry.lab.testCases.CheckoutPageValidator">
    <skipped/>
  </testcase> <!-- validateCheckOutPage -->
  <system-out/>
</testsuite> <!-- com.vcentry.lab.testCases.CheckoutPageValidator -->
